src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -40,6 +41,11 @@ export class MultiSelectComponent<T> implements OnInit, ControlValueAccessor {
 
   private isTouched: boolean = false
 
+  @HostListener('click', ['$event'])
+  protected stopEventPropagation(event: MouseEvent): void {
```

Comment:
It would be very nice if the first method of each class is the constructor. This makes it easy to always find the constructor (which is the first thing run in a classs) and it gives a nice separation between instance variables and methods.

Reply:
Fixed